spacy_llm/models/rest/openai/registry.py

```diff
@@ -25,7 +25,7 @@
 def openai_gpt_4_v2(
     config: Dict[Any, Any] = SimpleFrozenDict(temperature=_DEFAULT_TEMPERATURE),
     name: Literal[
-        "gpt-4", "gpt-4-0314", "gpt-4-32k", "gpt-4-32k-0314"
+        "gpt-4", "gpt-4-0314", "gpt-4-32k", "gpt-4-32k-0314", "gpt-4-1106-preview"
```

Comment:
Ugh I'm again wondering whether we should version bump `GPT-4` for this. The issue is this: we'll document that `"spacy.GPT-4.v2"` has this expanded literal, but users will actually have to upgrade spaCy for this model name to become available, right? So there will be two versions of `"spacy.GPT-4.v2"` that don't behave exactly the same.